### PR TITLE
LPS-129513 Apply Styles panel visual adjustments

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by sandrodw3** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
Backend part is fine :D ✅ 

I've suggested some possible changes in the front to Sandro over slack

**Comment by sandrodw3** [comment]
ci:test:relevant

**Comment by sandrodw3** [comment]
ci:test:stable

**Comment by sandrodw3** [comment]
ci:test:sf

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/config/constants/containerWidthTypes.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+export const CONTAINER_WIDTH_TYPES = {
+	fixed: 'fixed',
+	fluid: 'fluid',
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/config/constants/containerWidthTypes.js:17]: 🌟

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/fragment-configuration-fields/SelectField.js [hunk 2]
```diff
@@ -78,6 +84,7 @@ export const SelectField = ({disabled, field, onValueSelect, value}) => {
 };
 
 SelectField.propTypes = {
+	calssName: PropTypes.string,
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/fragment-configuration-fields/SelectField.js:87]: typo :P

Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/fragment-configuration-fields/SelectField.js]: Weeee

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/config/constants/containerWidthTypes.js (hunk 1 above)
Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/config/constants/containerWidthTypes.js:17]: 🌈

**Comment by sandrodw3** [comment]
ci:forward

**Comment by sandrodw3** [comment]
ci:stop

**Comment by sandrodw3** [comment]
Leaving this on hold until master is synced with Brian's and we can rebase and resolve some conflicts 🕯

**Comment by sandrodw3** [comment]
ci:forward

**Comment by sandrodw3** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
Hi @sandrodw3, could you update the json file used in integration tests?

**Comment by sandrodw3** [comment]
ci:forward

**Comment by sandrodw3** [comment]
ci:forward

**Comment by sandrodw3** [comment]
ci:forward

**Pull request closed without merging**